test/integration/recruiters.test.js

```diff
@@ -69,4 +82,80 @@ describe("Recruiters", function () {
         });
     });
   });
+
+  describe("GET /members/intro", function () {
+    it("Should return empty array if no recruiters data is found", function (done) {
+      chai
+        .request(app)
+        .get("/members/intro")
+        .set("cookie", `${cookieName}=${jwt}`)
+        .end((err, res) => {
+          if (err) {
+            return done(err);
+          }
+          expect(res).to.have.status(200);
+          expect(res.body).to.be.a("object");
+          expect(res.body.message).to.equal("Recruiters returned successfully!");
+          expect(res.body.recruiters).to.eql([]);
+
+          return done();
+        });
+    });
+
+    it("Get all the recruiters data in the database", function (done) {
+      recruiters.addRecruiterInfo(recruiterData, username).then(() => {
+        chai
+          .request(app)
+          .get("/members/intro")
+          .set("cookie", `${cookieName}=${jwt}`)
+          .end((err, res) => {
+            if (err) {
+              return done(err);
+            }
+            expect(res).to.have.status(200);
+            expect(res.body).to.be.a("object");
+            expect(res.body.message).to.equal("Recruiters returned successfully!");
+            expect(res.body.recruiters).to.be.a("array");
+            expect(res.body.recruiters).to.have.length.above(0);
+            expect(res.body.recruiters[0]).to.have.all.keys(
+              "company",
+              "first_name",
+              "last_name",
+              "designation",
+              "reason",
+              "email",
+              "currency",
+              "package",
+              "timestamp",
+              "package",
```

Comment:
Looks like the second **package** is a mistake here?

Reply:
Thanks for pointing it out. I will fix that